symfony/messenger/4.3/config/packages/messenger.yaml

```diff
@@ -0,0 +1,14 @@
+framework:
+    messenger:
+        # Uncomment the following line and choose a transport to collect messages that fail so you can retry them later
```

Comment:
We can now tweak this:

Uncomment this (and the failed transport below) to send failed messages to this transport for later handling.